samples/valueentity-counter/.gitignore

```diff
@@ -0,0 +1,2 @@
+user-function.desc
+package-lock.json
```

Comment:
Here we exclude package-lock.json, but this file is in Git already. What was the intention? 

Also, since I'm not familiar with JS stack, what is this file good for and what is the common practice in the JS world. Ignore or include in Git?

Maybe @katsutoxin can give us some advice?

Reply:
The `package-lock.json` should not be in `.gitignore`. 

> The package-lock.json file needs to be committed to your Git repository, so it can be fetched by other people, if the project is public or you have collaborators, or if you use Git as a source for deployments.

Source: https://nodejs.dev/learn/the-package-lock-json-file (which is maintained by the Node.js community)

Reply:
we are ignoring the sample `package-lock.json`(that is not available in git) and we are updating the `sdk` `package-lock.json`.

I'm ignoring the `lock` file to make sure that sdk dependencies are freshly resolved in CI for this project.

Reply:
> The goal of package-lock.json file is to keep track of the exact version of every package that is installed so that a product is 100% reproducible in the same way even if packages are updated by their maintainers.

In this very specific case, we want to bump and check the sdk with all transitive dependencies updated.